sdk/core/Azure.Core/src/Pipeline/HttpWebRequestTransport.cs

```diff
@@ -86,7 +86,7 @@ private async ValueTask ProcessInternal(HttpMessage message, bool async)
             // WebException is thrown in the case of .Abort() call
             catch (WebException) when (message.CancellationToken.IsCancellationRequested)
             {
-                message.CancellationToken.ThrowIfCancellationRequested();
+                throw new TaskCanceledException();
```

Comment:
Don't we need the token property here?

Reply:
I don't think it matters in this particular case. HttpClient doesn't set it correctly on .net framework so this behavior would be consistent with what we have right now.

Reply:
Worth adding a unit test to ensure that behavior never changes?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/13952/files#diff-637d8eae12196608f9581a0e65e40697R257

Reply:
I meant verify the token isn't set in case they change that at some point.  I don't think it's critical.